src/GitLink/Helpers/ProjectHelper.cs

```diff
@@ -29,7 +29,7 @@ public static class ProjectHelper
 
         static ProjectHelper()
         {
-            SolutionParserType = TypeCache.GetType("Microsoft.Build.Construction.SolutionParser");
+            SolutionParserType = TypeCache.GetType("Microsoft.Build.Construction.SolutionParser, Microsoft.Build, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
```

Comment:
I think the other line takes care of that. The runtime should load any version available without the version specified. If you specify the version (and public key), it will only work with that version.